inference_schema/schema_decorators.py

```diff
@@ -54,7 +54,7 @@ def decorator_input(user_run, instance, args, kwargs):
                                     'is not in the decorated function.'.format(param_name))
                 param_position = arg_names.index(param_name)
                 args[param_position] = _deserialize_input_argument(args[param_position], param_type, param_name)
-            elif param_name not in kwargs.keys() and optional:
+            elif (param_name not in kwargs.keys() and optional) or (kwargs[param_name] is None and optional):
```

Comment:
Suggest to change `kwargs[param_name] is None` to `not kwargs[param_name]`. That should handle other edge cases of "None" like empty string or empty list

Reply:
The whole thing can probably also be simplified with `elif (optional and (param_name not in kwargs.keys() or not kwargs[param_name]))